extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergWriteInstructions.java

```diff
@@ -41,6 +41,15 @@ public static Builder builder() {
      */
     public abstract List<String> partitionPaths();
 
+    /**
+     * Whether to apply the {@link org.apache.iceberg.Table#sortOrder()} of the iceberg table. If true, the tables will
+     * be sorted before writing to the iceberg table. By default, this is false.
+     */
+    @Value.Default
+    public boolean applySortOrder() {
+        return false;
+    }
```

Comment:
I think we may want / need to be more explicit and allow the user to specify the specific SortOrder they want to use.